specification/cognitiveservices/data-plane/QnAMaker/readme.md

```diff
@@ -0,0 +1,99 @@
+# Cognitive Services QnAMaker SDK
+
+> see https://aka.ms/autorest
+
+Configuration for generating QnAMaker SDK.
+
+The current release is `release_4_0`.
+
+``` yaml
+
+tag: release_4_0
+add-credentials: true
+openapi-type: data-plane
+```
+# Releases
+
+### Release 4.0
+These settings apply only when `--tag=release_4_0` is specified on the command line.
+
+``` yaml $(tag) == 'release_4_0'
+input-file: stable/v4.0/QnAMaker.json
+```
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+``` yaml $(swagger-to-sdk)
+swagger-to-sdk:
+  - repo: azure-sdk-for-python
+  - repo: azure-sdk-for-java
+  - repo: azure-sdk-for-go
+  - repo: azure-sdk-for-js
+  - repo: azure-sdk-for-node
+  - repo: azure-sdk-for-ruby
+    after_scripts:
+      - bundle install && rake arm:regen_all_profiles['azure_cognitiveservices_qnamaker']
+```
+
+
+## CSharp Settings
+These settings apply only when `--csharp` is specified on the command line.
+``` yaml $(csharp)
+csharp:
+  sync-methods: None
+  license-header: MICROSOFT_MIT_NO_VERSION
+  azure-arm: false
+  namespace: Microsoft.Azure.CognitiveServices.QnAMaker
```

Comment:
namespace should be current `Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker`